website/assets/scss/components/_schema-renderer.scss

```
@@ -297,6 +297,34 @@ $schema-max-depth: 8;
+.sr-const-value {
+  display: inline-block;
+  padding: 0.15rem 0.4rem;
+  border: 1px solid var(--bs-border-color);
+  border-radius: 4px;
+  background-color: var(--bs-body-bg);
+  color: var(--bs-body-color);
+  font-size: 0.6875rem;
+  font-weight: 600;
+  line-height: 1.35;
+  white-space: normal;
+  word-break: break-word;
```

Stylelint (17.11.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)